lib/code_corps/policy/policy.ex

```diff
@@ -30,6 +30,8 @@ defmodule CodeCorps.Policy do
   defp can?(%User{} = user, :update, %Comment{} = comment, %{}), do: Policy.Comment.update?(user, comment)
   defp can?(%User{} = user, :create, %Organization{}, %{}), do: Policy.Organization.create?(user)
   defp can?(%User{} = user, :update, %Organization{} = organization, %{}), do: Policy.Organization.update?(user, organization)  
+  defp can?(%User{} = current_user, :update, %User{} = user, %{}), do: Policy.User.update?(user, current_user)
+  defp can?(%User{}, _action, %User{} = _user, %{}), do: true
```

Comment:
Remind me again, why is this one needed? It seems unsafe, as it would technically allow any user to perform  any action on any other user. 

If you meant to copy-paste the one from line 46 below, then the third argument would be nil, but that means you'd also have to update the spec to allow nil, not just `User.t` as the third argument.

However, I don't think the catch all from below is strictly necessary when not using canary, so could you double check?

Reply:
Yep my bad.  Good call!